docs/structure-samples/csharp/csharp.md

```diff
@@ -0,0 +1,53 @@
+## C# and .NET project folder structure
+
+The following guidelines and the folders included, represent the conventional structure for a C#/.NET standard application.
+
+```bash
+~root
+│
+├── .devcontainer.json          # Moved to top
+├── .github/
+│   └── workflows/
+│       ├── workflow1.yml
+│       ├── workflow2.yml
+│       └── ...                     
+│
+├── infra/
+│   ├── main.bicep
+│   ├── main.parameters.bicep
+│   ├── abbreviations.json
+│   └── ...                         
+│
+├── src/
+│   ├── ProjectName.Core/
```

Comment:
besides the files required by AZD, eg. /infra, azure.yml, why we want to standarize the source folder structure under /src? It should vary depending on app business logic.

Reply:
The inside of the folder we won't standarize. We want to make sure the source is under `/src`. Those file names are examples.

Reply:
Every sample is under /src.